submit/controllers/ui/new/authorship.py

```diff
@@ -35,8 +36,6 @@ def authorship(method: str, params: MultiDict, session: Session,
                submission_id: int, **kwargs) -> Response:
     """Handle the authorship assertion view."""
     submitter, client = user_and_client_from_session(session)
-
-    # Will raise NotFound if there is no such submission.
```

Comment:
Isn't this still the case?

Reply:
Yes. In submit/utils.py:

        try:
            submission, submission_events = events.load(submission_id)
        except events.exceptions.NoSuchSubmission as e:
            raise NotFound('No such submission.') from e